test_auth.py

```diff
@@ -93,6 +93,16 @@ def test_site_admin():
         warnings.warn(UserWarning("OPA_URL is not set"))
 
 
+def test_user_email():
+        """
+        If OPA is present, check to see that the user's email is returned.
+        """
+        if OPA_URL is not None:
```

Comment:
Might also want to test OPA_SECRET as well, since it is passed along.

Reply:
These tests are pretty much just to test internal changes; I think that it's very unlikely that you'd have OPA_URL set but not OPA_SECRET.